scripts/nice2/helpers.sh

```diff
@@ -0,0 +1,217 @@
+#!/bin/bash
+set -e
+
+function loadEnvs() {
+  export $(cat .env | sed 's/#.*//g' | xargs)
+}
+
+loadEnvs
+
+BackupFile=./scripts/nice2/nice2_dump.psql
+PostgresPort=5432
+
+DATABASE_SERVERNAME="${HIBERNATE_MAIN_SERVERNAME:-localhost}"
+DATABASE_USER="${HIBERNATE_MAIN_USER:-nice}"
+DATABASE_PASSWORD="${HIBERNATE_MAIN_PASSWORD:-nice}"
+DATABASE_DATABASENAME="${HIBERNATE_MAIN_DATABASENAME:-test_cypress}"
+DATABASE_SSLMODE="${HIBERNATE_MAIN_SSLMODE:-disable}"
+
+function getNice2Folder() {
+  CurrentNiceVersion=$(cat ./nice-current-version.txt)
+  PropertiesNiceVersion=$(echo "$CurrentNiceVersion" | sed -e 's/\.0\s*$//g')
+
+  for Folder in $NICE2_REPO_BASE_PATH/*
+  do
+    PropertiesFile="$Folder/boot/src/main/resources/ch/tocco/nice2/boot/impl/default.properties"
+    if [[ -f "$PropertiesFile" ]]; then
+      Properties=$(cat $PropertiesFile)
+      if [[ "$Properties" == *"$PropertiesNiceVersion"* ]]; then
+        Nice2Folder="$Folder"
+      fi
+    fi
+  done
+
+  echo "$Nice2Folder"
+}
+
+function waitFor() {
+    timeout=1000
+    until [ $timeout -le 0 ] || ("$@" &> /dev/null); do
+        echo waiting for "$@"
+        sleep 10
+        timeout=$(( timeout - 1 ))
+    done
+    if [ $timeout -le 0 ]; then
+        return 1
+    fi
+}
+
+function postgresIsReady() {
+    pg_status=$(pg_isready -d ${DATABASE_DATABASENAME} -h ${DATABASE_SERVERNAME} -p $PostgresPort -U ${DATABASE_USER} | grep -o "accepting connections")
+    if [ "${pg_status}" = "accepting connections" ]; then
+        return 0;
+    else
+        return 1;
+    fi
+}
+
+function nice2IsReady() {
+    status_code=$(curl --write-out '%{http_code}' --silent --output /dev/null http://localhost:8080/status-tocco)
+    if [[ "$status_code" -ne 200 ]] ; then
+        return 1;
+    else
+        return 0;
+    fi
+}
+
+function emptyDB() {
+  echo "drop database '$DATABASE_DATABASENAME'"
+
+  psql -v ON_ERROR_STOP=1 -h ${DATABASE_SERVERNAME} -p $PostgresPort <<EOF
+    DROP DATABASE IF EXISTS $DATABASE_DATABASENAME;
+EOF
+
+  echo "create database '$DATABASE_DATABASENAME'"
+
+  psql -v ON_ERROR_STOP=1 -h ${DATABASE_SERVERNAME} -p $PostgresPort <<EOF
+    CREATE DATABASE $DATABASE_DATABASENAME WITH OWNER $DATABASE_USER;
+EOF
+
+  psql -v ON_ERROR_STOP=1 -h ${DATABASE_SERVERNAME} -p $PostgresPort -d ${DATABASE_DATABASENAME} -U $DATABASE_USER <<EOF
+    CREATE EXTENSION IF NOT EXISTS lo;
+    CREATE EXTENSION IF NOT EXISTS pg_trgm;
+    CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+EOF
+}
+
+
+function setupCypressUser() {
+  echo "insert principal '$CYPRESS_USER'"
+
+  psql -v ON_ERROR_STOP=1 -h ${DATABASE_SERVERNAME} -p $PostgresPort -U ${DATABASE_USER} -d ${DATABASE_DATABASENAME} <<EOF
+BEGIN;
+
+INSERT INTO nice_principal
```

Comment:
This scripts contains variables. The script in the backend doesn't because the hashes have been checked into the repo.
Can I extract it into a separate file then?